spec/lib/merge_supporters_spec.rb

```diff
@@ -272,12 +272,14 @@
       expect(supporter_1.reload.merged_into).to eq(supporter_2.merged_into)
     end
 
+    it 'creates a supporter.created object event' do
+      expect { result }.to change{ ObjectEvent.where(event_type: 'supporter.created').count }.by 1
+    end
```

Comment:
This is the only functional change to this spec. I saw an opportunity to refactor a tiny bit and took it lol